src/saml2/config.py

```diff
@@ -97,6 +97,7 @@
     "sp_type",
     "sp_type_in_metadata",
     "requested_attributes",
+    "node_country",
```

Comment:
Do both SPs and IdPs need to define a `node_country`? What does it mean in each case?

Reply:
According to the saml-message-format v1.2 document, in the metadata examples, both the eIDAS Service (IdP) and the Connector (SP) declare NodeCountry in their metadata.

> What does it mean in each case?

What do you mean "what does it mean" ? :stuck_out_tongue:

Reply:
Both the eIDAS Service and the Connector are proxies; they both have an IdP and an SP interface.

Quoting, "eIDAS SAML Message Format v.1.2 Final.pdf", section "4.2. NODE COUNTRY":
> For indicating which Member State or international organisation is responsible for an eIDAS-Node (serving this Member State’s eID means, this Member State’s or international organisation’s relying parties, respectively), the element `<eidas:NodeCountry>` (see Section 5.2) MUST be present in the `<md:Extensions>` element of SAML metadata. In detail, if the eIDAS-Node acts as a eIDAS-Service, the `<eidas:NodeCountry>` MUST be present in the `<md:Extensions>` element of the `<md:IDPSSODescriptor>`. If the eIDAS-Node acts as a eIDAS-Connector, the `<eidas:NodeCountry>` MUST be present in the `<md:Extensions>` element of the `<md:SPSSODescriptor>`.
>
> The SAML authentication request MUST NOT contain a `<eidas:NodeCountry>` element. The value of the `<eidas:NodeCountry>` element MUST be the Nationality Code of the SP country or international organization2 in _ISO 3166-1 alpha-2_ format.